recipes/bezier/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "bezier" %}
+{% set version = "0.2.1" %}
+{% set compress_type = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "d9540e9af445b309e5eb89f4d6666d9f1ad755bb69c7cde3d7f814cbcbbf9144" %}
+{% set build_num = "0"%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34
```

Comment:
The extension module has been added **after** the `0.3.0` release (i.e. in unreleased commits) so it's definitely not in the `0.2.1` release. But thanks for noticing :grin: